cfb-mode/Cargo.toml

```diff
@@ -12,6 +12,7 @@ categories = ["cryptography", "no-std"]
 [dependencies]
 stream-cipher = "0.3"
 block-cipher-trait = "0.6"
+zeroize = { version = "*", optional = true }
```

Comment:
Would probably prefer locking to a particular version here. As it were I might try to put another one out today... will follow up after I do

Reply:
Also note: I'd like to get to 1.0 soon, but I want to verify the generated assembly appears to do the right thing on more platforms than just x86(-64).

Reply:
You could maybe write a test that uses unsafe to turn a static pointer into one with a lifetime, then initialize it with non-zero values, then drop it and test that the data got zeroed out.

It's a bit more advanced than my current rust skill level, but I could look into it on a weekend maybe.

Reply:
Just a quick note: after reading [this thread](https://github.com/rust-lang/unsafe-code-guidelines/issues/33) and seeing [this change to the Rust documentation](https://github.com/rust-lang/rust/pull/60972), I'm feeling *much* better about Zeroize's current semantics and plan on dramatically reducing the length of the explanation along with much of the scary language

It seems much of what is in there is no longer aligned with the opinions of the Unsafe Code WG, which is to say the behavior is defined and, theoretically, a safe abstraction.

That said, after the 0.8 release I am still tweaking things like trait bounds and the procedural macro, so I'd love to get feedback from people integrating it (I am also attempting to integrate it into curve25519-dalek).